contracts/lib/Organization.sol

```diff
@@ -197,20 +254,7 @@ contract Organization is OrganizationInterface, IsMemberInterface {
         onlyOwnerOrAdmin
         returns (uint256 remainingBlocks_)
     {
-        require(
-            _worker != address(0),
-            "Worker address cannot be null."
-        );
-
-        require(
-            _expirationHeight > block.number,
-            "Expiration height must be in the future."
-        );
-
-        workers[_worker] = _expirationHeight;
-        remainingBlocks_ = _expirationHeight.sub(block.number);
-
-        emit WorkerSet(_worker, _expirationHeight, remainingBlocks_);
+        remainingBlocks_ = setWorkerInternal(_worker, _expirationHeight);
     }
 
     /**
```

Comment:
I would suggest to remove isUnset variable at all. This costs gas and can be checked by caller if needed.

Reply:
If you refer to `unsetWorker`: ✅ done
Otherwise please give context I am not sure this comment belongs in this line.

Reply:
My bad, I was referring to remainingBlocks parameter.

Reply:
ok. that's tracked in https://github.com/OpenSTFoundation/mosaic-contracts/pull/547/files#r242095075